php/ext/google/protobuf/message.c

```diff
@@ -250,15 +268,11 @@ static int Message_compare_objects(zval* m1, zval* m2) {
 static int Message_has_property(zend_object* obj, zend_string* member,
                                 int has_set_exists, void** cache_slot) {
   Message* intern = (Message*)obj;
-  const upb_FieldDef* f = get_field(intern, member);
+  const upb_FieldDef* f = lookup_field(intern, member);
 
   if (!f) return 0;
 
-  if (!upb_FieldDef_HasPresence(f)) {
-    zend_throw_exception_ex(
-        NULL, 0,
-        "Cannot call isset() on field %s which does not have presence.",
-        upb_FieldDef_Name(f));
+  if (upb_FieldDef_IsOptional(f) && !upb_FieldDef_HasPresence(f)) {
```

Comment:
Any particular reason why you're removing the error message above? 

I am not sure if it's appropriate or not, but I'd rather not change behavior unless we have a specific reason for doing so (and this isn't mentioned in issue #20648)

Reply:
Sorry I only notice that one while implementing the fix / test. 
I can open another issue if you think that makes sense.  but without it isset is still unusable in many scenarios..

That will prevent using `isset($message->optional_string)` on a existing field. 
```
1) GeneratedClassTest::testIsset
Exception: Cannot call isset() on field optional_string which does not have presence.

protobuf/php/tests/GeneratedClassTest.php:69
```

I don't think `isset` should ever throw..

Reply:
Hmm that's a good point. There's nowhere in PHP where calling `isset` should throw an exception.